core/src/main/scala/za/co/absa/spline/core/harvester/DataLineageBuilder.scala

```diff
@@ -62,11 +75,20 @@ class DataLineageBuilder(logicalPlan: LogicalPlan, executedPlanOpt: Option[Spark
           if (parentBuilder != null) parentBuilder += curBuilder
 
           if (maybeExistingBuilder.isEmpty) {
-            val newNodesToProcess =
-              writeCommandParser.
+
+            //try to find all possible commands for traversing- save to filesystem, saveAsTable, JDBC
+            val writes = writeCommandParser.
```

Comment:
Can you really have multiple writes in single query?

Reply:
No, you cannot

Reply:
ok, the comment is bit confusing then maybe even the code could be written more in line with that.

Reply:
Yes, it is. I wanted to say that it handles all types of commands, but there is only one at time.